dotnet/src/SemanticKernel.Abstractions/Config/HttpHandlerFactory{THandler}.cs

```diff
@@ -0,0 +1,15 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+
+using System;
+using System.Net.Http;
+using Microsoft.Extensions.Logging;
+
+namespace Microsoft.SemanticKernel.Config;
+
+public abstract class HttpHandlerFactory<THandler> : IDelegatingHandlerFactory where THandler : DelegatingHandler
```

Comment:
Maybe this is all better in Microsoft.SemanticKernel.Http or similar?

Reply:
Agree, moved to Http namespace